text/dependencies/rfcs/0000-dependency-management-phase-one.md

```diff
@@ -0,0 +1,302 @@
+# Dependency Management Phase 1: Adopting Federated Model RFC
+
+## Proposal
+
+The buildpacks should adopt a federated dependency management strategy. This
+will involve all of the buildpacks that provide depdendencies (such as the [Go
+Dist CNB](https://github.com/paketo-buildpacks/go-dist) or the [MRI
+CNB](https://github.com/paketo-buildpacks/mri)) becoming the owners of
+dependency-specific logic that currently resides in the
+[dep-server](https://github.com/paketo-buildpacks/dep-server). This includes
+code for discovering new versions, retrieving dependencies, and compiling the
+dependencies. As a result, maintainers will make/maintain dependency-related
+decisions for the dependencies of relevant buildpacks.  Additionally,
+buildpacks should use dependencies directly from upstream hosting (source or
+binary) whenever possible, to cut down on the number of dependencies that are
+uncessarily compiled from source.
+
+## Motivation
+
+Per top-level [RFC
+0000](https://github.com/paketo-buildpacks/rfcs/blob/dependency-management-top-level/text/dependencies/rfcs/0000-dependency-management-overview.md),
+the current dependency management system employed
+in the project leads to a number of painpoints, predominantly around
+maintainability for Dependencies maintainers, and lack of extensibility by
+compiling dependencies for Ubuntu 18.04. In order to ensure the Paketo
+Buildpacks project can continue to grow and introduce new buildpacks,
+dependencies, and stacks, business logic that pertains to specific dependencies
+should be the responsibility of the relevant buildpack.
+
+## Detailed Explanation
+
+The outlined proposal contains a number of changes to the current system which
+should be standardized in order to fit together with generalized Github Actions
+workflows down the line.
+
+### Background
```

Comment:
I think it's worth detailing the current workflow for the Java buildpacks as well as the other language families either in background or as an alternative. What tradeoffs would the Java buildpacks have to make to move to this model? Have we considered the pros/cons of adopting more aspects of the Java buildpack workflows in new proposal?

Can we even standardize across all buildpacks without first standardizing the metadata schema?

cc @dmikusa-pivotal